config/initializers/health_monitor.rb

```diff
@@ -7,7 +7,11 @@
       c.url = Blacklight.default_index.connection.uri.to_s
     end
 
+    config.add_custom_provider(AeonStatus)
```

Comment:
Should all of these new providers be marked as critical? (i.e., should the load balancer not serve the app if any of them are down?) I would argue that all of the newly added ones should report, but not be considered critical.

Reply:
Made the change. Couldn't get it to work in the config, but was able to make them non-critical in the custom providers.